ansible_collections/arista/avd/plugins/plugin_utils/eos_designs_facts.py

```diff
@@ -1830,7 +1830,7 @@ def mlag_peer_mgmt_ip(self):
         return None
 
     @cached_property
-    def overlay_routing_protocol_address_family(self):
+    def _overlay_routing_protocol_address_family(self):
```

Comment:
Please revert this change, since it was already there without the underscore, meaning it ends up as `switch.overlay_routing_protocol_address_family` which may be used elsewhere in 3.8.

Reply:
If we remove underscore "_" then we get the following error:
  An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ansible_collections.arista.avd.plugins.plugin_utils.errors.errors.AristaAvdError: 'overlay_routing_protocol_address_family: ipv6' is only supported in combination with 'underlay_ipv6: True' and 'underlay_rfc5549: True'

Reply:
Yeah because changing this will break that check overall even for other device types.

Reply:
Done :)